src/GslibIO.jl

```diff
@@ -155,4 +196,58 @@ function save(file::File{format"GSLIB"}, sdata::AbstractData)
        propnames=vars)
 end
 
+# low level function for saving `data` to a legacy GSLIB format using `varnames` as variable names
+function save_legacy(filename::AbstractString, data::AbstractMatrix, varnames::NTuple, header::AbstractString, na)
```

Comment:
```suggestion
function save_legacy(filename::AbstractString, data::AbstractMatrix,
                                   varnames::NTuple, header::AbstractString, na)
```

Reply:
GitHub has some poor indentation in these suggestion comments. Please adjust the indentation locally if it can't be adjusted here on the web browser.